third_party/nccl.BUILD

```diff
@@ -43,6 +43,44 @@ cc_library(
         "-Iexternal/nccl_archive/src",
         "-O3",
     ] + cuda_default_copts(),
+    linkopts = select({
+        ":android": [
```

Comment:
//tensorflow:android can be used instead, and remove the defines below?   Although I see that curl.BUILD does this as well, so I think it would be okay to keep it for now, and I can edit both after.  It's up to you if you want to change it again, I'll launch testing on this version.